docs/walkthrough.md

```diff
@@ -50,132 +51,92 @@ When we create this `ClusterServiceBroker` resource, the service catalog control
 
 The controller created a `ClusterServiceClass` for each service that the UPS broker
-provides. We can view the `ClusterServiceClass` resources available in the cluster by
-executing:
+provides. We can view the `ClusterServiceClass` resources available by executing:
 
 ```console
-$ kubectl get clusterserviceclasses -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
-NAME                                   EXTERNAL NAME
-4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468   user-provided-service
+$ svcat get classes
+                NAME                        DESCRIPTION                         UUID
++-----------------------------------+-------------------------+--------------------------------------+
+  user-provided-service               A user provided service   4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
+  user-provided-service-single-plan   A user provided service   5f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
 ```
 
-**NOTE:** The above command uses a custom set of columns.  The `NAME` field is
-the Kubernetes name of the `ClusterServiceClass` and the `EXTERNAL NAME` field is the
-human-readable name for the service that the broker returns.
-
 The UPS broker provides a service with the external name
 `user-provided-service`. Run the following command to see the details of this
 offering:
 
 ```console
-$ kubectl get clusterserviceclasses 4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468 -o yaml
-apiVersion: servicecatalog.k8s.io/v1beta1
-kind: ClusterServiceClass
-metadata:
-  creationTimestamp: 2017-11-01T14:12:29Z
-  name: 4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  resourceVersion: "4"
-  selfLink: /apis/servicecatalog.k8s.io/v1beta1/clusterserviceclasses/4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  uid: b1e764ba-bf0e-11e7-9e29-0242ac110004
-spec:
-  bindable: true
-  clusterServiceBrokerName: ups-broker
-  description: A user provided service
-  externalID: 4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
-  externalName: user-provided-service
-  planUpdatable: false
-status:
-  removedFromBrokerCatalog: false
+$ svcat describe class user-provided-service
+  Name:          user-provided-service
+  Description:   A user provided service
+  UUID:          4f6e6cf6-ffdd-425f-a2c7-3c9258ad2468
+  Status:        Active
+  Tags:
+  Broker:        ups-broker
+
+Plans:
+   NAME           DESCRIPTION
++---------+-------------------------+
+  default   Sample plan description
+  premium   Premium plan
 ```
 
 Additionally, the controller created a `ClusterServicePlan` for each of the
 plans for the broker's services. We can view the `ClusterServicePlan`
 resources available in the cluster by executing:
 
 ```console
-$ kubectl get clusterserviceplans -o=custom-columns=NAME:.metadata.name,EXTERNAL\ NAME:.spec.externalName
-NAME                                   EXTERNAL NAME
-86064792-7ea2-467b-af93-ac9694d96d52   default
+$ svcat get plans
+   NAME                   CLASS                       DESCRIPTION                         UUID
```

Comment:
Wondering to myself whether we should even show the UUID column unless a user wants to see it (not feedback on this PR :))

Reply:
I have wanted to hide UUID since the beginning! :trollface:  I'll submit a PR for that separately.

Reply:
I was thinking that we wouldn't show UUID in the table formatted output, and only show it for describe. So it's still accessible from svcat, but not one of the "chosen" fields that are printed all the time.

Reply:
I've started an issue about UUID and provide example output of what I'm proposing.

https://github.com/kubernetes-incubator/service-catalog/issues/1804